sphinx/util/osutil.py

```diff
@@ -235,7 +235,7 @@ def abspath(pathdir):
         try:
             pathdir = pathdir.decode(fs_encoding)
         except UnicodeDecodeError:
-            raise UnicodeDecodeError('multibyte filename not supported on '
+            raise UnicodeDecodeError('multibyte filename not supported on '  # type: ignore
```

Comment:
@shimizukawa It seems this error handling is wrong. Surely `UnicodeDecodeError` excepts 5 arguments for instantiation. Could you send a PR for this please?
refs: df3643722f